### PR TITLE
Add contributing guidelines.

### PR DESCRIPTION
**Pull request by alunduil**

### CODE REVIEW

**Comment by alunduil** [comment]
Blocked by https://github.com/macisamuele/language-formatters-pre-commit-hooks/pull/134

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+1. If not using VS Code, run `poetry install`
+1. If not using VS Code, run `poetry shell`
+1. Run `pre-commit install` in your local checkout
+1. Make the changes in your fork
+1. Test your changes with `pytest`
+1. Send your changes in a new pull request describing why you want to make that
+   change
+
+For small changes, feel free to use the in GitHub editor or skip running the
+tests locally (they'll be double checkeed in the pull request automatically).
```
Comment (jjalfaro9) [inline CONTRIBUTING.md:53]: checkeed -> checked

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+you have an interest in being an owner of this project, simply ask.
+
+## Community
+
+* [Current Contributors](https://github.com/alunduil/zfs-replicate/graphs/contributors)
+* Responses to issues and pull requests should be expected within two weeks of submission.
+
+[First Timers Only]: https://www.firsttimersonly.com/
+[good first issues]: https://github.com/alunduil/zfs-replicate/issues?q=is%3Aissue+label%3A%22good+first+issue%22+is%3Aopen
+[How to Contribute to an Open Source Project on GitHub]: https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github
```
Comment (jjalfaro9) [inline CONTRIBUTING.md:98]: https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github ->  https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+1. A link to a paste with your `/etc/os-release` file (or equivalent for Mac or
+   Windows)
+1. What command did you run (please remove passwords or secrets if present)
+1. What happened?
+1. What did you expect to happen?
+
+## How to Suggest a Feature or Enhancement
+
+zfs-replicate aims to be a simple to use wrapper around the ssh and zfs command
+line tools.  Other tools exist for more complex interactions than ship my
```
Comment (jjalfaro9) [inline CONTRIBUTING.md:76]: `Other tools exist for more complex interactions than ship my snapshots from here to there.`




what does this mean?

**Review by jjalfaro9 — COMMENTED** [review]
couple of suggestions & one question. other than that, this seems good.

File: CONTRIBUTING.md (hunk 3 above)
Comment (alunduil) [reply CONTRIBUTING.md:76]: Basically a recap of the alternatives from the README.  Let me see if I can simplify this language a bit.

**Review by jjalfaro9 — APPROVED** [review]
lgtm, vamosss!

**Pull request merged**